rolling-update-kubernetes/src/main/resources/reference.conf

```diff
@@ -34,6 +34,10 @@ akka.rollingupdate.kubernetes {
     pod-name = ""
     pod-name = ${?KUBERNETES_POD_NAME}
 
+    # Annotations to check to determine the revision.  The default is suitable for "vanilla"
+	# Kubernetes Deployments, but other CI/CD systems may set a different annotation.
```

Comment:
Is this indentation a leftover? And does it use a tab, instead of space currently?